### ISSUE TITLE
Lines of code present in PyPi package but not in documentation (1.3.2)

### ISSUE THREAD

**Issue by THinnerichs** [rusty1s/pytorch_geometric#930]
## 🐛 Bug

Not really a bug, but inconsistent state of documentation and official build. Both confirmed version 1.3.2.

When reading the PyPi files in ```pytorch_geometric/data/batch.py ``` the following lines are present:

```
...
else:
    raise ValueError('Unsupported attribute type')
...
```
while checking the validity of the datatype of the graph input. These are leading to the suspected behaviour, and are thus not really a bug. However, these lines are neither present in github, nor in the official documentation. Both are the same version 1.3.2. 

Thus, you may want to recompile/reupload/update the PyPi package version of pytorch_geometric.

**Comment by rusty1s** [comment]
Thanks for reporting. I believe this is fixed with version 1.4.1!

**Comment by THinnerichs** [comment]
Thanks a lot! :)
